Sources/Plasma/PubUtilLib/plAvatar/plArmatureMod.cpp

```diff
@@ -826,10 +826,10 @@ void plArmatureMod::SpawnAt(int spawnNum, double time)
     w2l.RemoveScale();
     ci->SetTransform(l2w, w2l);
     ci->FlushTransform();
-    
-    if (plVirtualCam1::Instance())
-        plVirtualCam1::Instance()->SetCutNextTrans();
-    
+
+    if (IsLocalAvatar() && plVirtualCam1::Instance())
```

Comment:
Would you explain for the n00bs why the `IsLocalAvatar()` check was added? And now that the non-transitional cut is being used here, do other places still use the transitional camera cut, or is `SetCutNextTrans` obsolete now?

Reply:
This ensures that, if a remote player is panic respawned, his/her respawning will not taint your client's camera state. `SetCutNextTrans()` is still used by `plWarpMsg` and the first to third person transition from what I can tell.